src/style.css

```diff
@@ -0,0 +1,132 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+
+.page_body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card_block {
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+  
+.card_block__first_element {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card_block__img {
+  width: 160px;
+  height: 134px;
+  padding-bottom: 40px;
+}
+
+.card_block__text {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  padding-bottom: 4px;
+}
+
+.card_block__second_element {
+  display: flex;
+  flex-direction: column;
+}
+
+.card_block__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-bottom: 16px;
+}
+
+.card_block__status {
+  display: flex;
+  flex-direction: column;
+}
+
+.card_block__evaluation {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 24px;
+}
+
+.card_block__star {
+  display: flex;
+  padding-left: 2px;
+}
+
+.card_block__star_item {
+  --size_star: 16px;
+  height: var(--size_star);
+  width: var(--size_star);
+  background: url(./images/star.svg)
+    no-repeat 0;
+  padding-right: 4px;
+}
+
+.card_block__star_item--active:nth-child(-n+4),
+.card_block__star_item--active:nth-child(n):hover {
+  background: url(./images/star-active.svg)
+    no-repeat 0;
+}
+
+.card_block__reviews {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card_block__price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card_block__price_text {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card_block__cost {
+  display: flex;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
looks like we reuse colors a lot
lets pass it to variables

Reply:
isn't fixed